src/Features/CSharp/Portable/InlineHints/CSharpInlineParameterNameHintsService.cs

```diff
@@ -91,5 +93,17 @@ protected override bool IsIndexer(SyntaxNode node, IParameterSymbol parameter)
         {
             return node is BracketedArgumentListSyntax;
         }
+
+        protected override string GetArgumentName(SyntaxNode argument, ISyntaxFactsService syntaxFacts)
```

Comment:
it's not really an argument name.  that implies something like `x: ...`.  What this is is more about getitng the Expression of an Argument and checking if it is an IdentifierName.  All that can be done in the base type.

Reply:
right, poor naming. I can move everything to the base type